content/home.md

```diff
@@ -40,6 +40,12 @@ Created by [Getty](https://www.getty.edu), free to use.
 
 <div class="action-button paired-button">
 
+[Demo Site](http://quire.getty.edu/demo/)
```

Comment:
I think it works here, but we should also prioritize updating the figures pages. Except for rotations, they're all TK

Reply:
@geealbers I removed the demo from the homepage since it still needs work and I don't want to hold this PR up.